### ISSUE TITLE
error when using int8 with yolov5 deepstream

### ISSUE THREAD

**Issue by NamBK99** [marcoslucianops/DeepStream-Yolo#453]
ERROR: Failed to create network using custom network creation function
ERROR: Failed to get cuda engine from custom library API
0:00:03.198190868 31161     0x1aeefcc0 ERROR                nvinfer gstnvinfer.cpp:676:gst_nvinfer_logger:<primary-inference> NvDsInferContext[UID 1]: Error in NvDsInferContextImpl::buildModel() <nvdsinfer_context_impl.cpp:2022> [UID = 1]: build engine file failed
0:00:03.247502027 31161     0x1aeefcc0 ERROR                nvinfer gstnvinfer.cpp:676:gst_nvinfer_logger:<primary-inference> NvDsInferContext[UID 1]: Error in NvDsInferContextImpl::generateBackendContext() <nvdsinfer_context_impl.cpp:2108> [UID = 1]: build backend context failed
0:00:03.247592911 31161     0x1aeefcc0 ERROR                nvinfer gstnvinfer.cpp:676:gst_nvinfer_logger:<primary-inference> NvDsInferContext[UID 1]: Error in NvDsInferContextImpl::initialize() <nvdsinfer_context_impl.cpp:1282> [UID = 1]: generate backend failed, check config file settings
0:00:03.247653682 31161     0x1aeefcc0 WARN                 nvinfer gstnvinfer.cpp:898:gst_nvinfer_start:<primary-inference> error: Failed to create NvDsInferContext instance
0:00:03.247684531 31161     0x1aeefcc0 WARN                 nvinfer gstnvinfer.cpp:898:gst_nvinfer_start:<primary-inference> error: Config file path: config_infer_primary_yoloV5.txt, NvDsInfer Error: NVDSINFER_CONFIG_FAILED
Error: gst-resource-error-quark: Failed to create NvDsInferContext instance (1): /dvs/git/dirty/git-master_linux/deepstream/sdk/src/gst-plugins/gst-nvinfer/gstnvinfer.cpp(898): gst_nvinfer_start (): /GstPipeline:pipeline0/GstNvInfer:primary-inference:
Config file path: config_infer_primary_yoloV5.txt, NvDsInfer Error: NVDSINFER_CONFIG_FAILED
Exiting app

**Comment by marcoslucianops** [comment]
Are following those steps: https://github.com/marcoslucianops/DeepStream-Yolo/blob/master/docs/INT8Calibration.md ?

**Comment by NamBK99** [comment]
yep. I followed this, but i can't run with int8. I even got the calib.table not found error, so the program used mode fp16 instead of int8

**Comment by GDbbq** [comment]
I had the same problem. I'm guessing it's something at the code level. The yolov5 model quantified by trtexec pqt can run normally, but the accuracy loss is serious. In addition, I found that the fp32 model and fp16 model are basically the same speed, 190fps. int8 model only 330fps.
![image](https://github.com/marcoslucianops/DeepStream-Yolo/assets/86715812/6acdd26d-1754-4044-8acd-4e79fad7f420)

**Comment by GDbbq** [comment]
nvidia device:  1080ti
![image](https://github.com/marcoslucianops/DeepStream-Yolo/assets/86715812/21d9c4da-b15b-4ec9-9419-bbaf73f4fb01)

**Comment by IronmanVsThanos** [comment]
> yep. I followed this, but i can't run with int8. I even got the calib.table not found error, so the program used mode fp16 instead of int8

have u solved this problem？
